btree/btree.c

```diff
@@ -6368,6 +6382,7 @@ static void btree_ut_kv_oper_thread_handler(struct btree_ut_thread_info *ti)
 			for (i = 1; i < ARRAY_SIZE(value); i++)
 				value[i] = value[0];
 
+			M0_SET0(&kv_op);
```

Comment:
Is this needed ? The reason is if similar code is present in Motr where the m0_btree_op is not initialized then will we crash?

Reply:
While debugging this crash, had doubts that we might be passing uninitialized `kv_op` to kv operations and it could have garbage data which might be causing this issue. Hence initialized it as a safety measure. But yes, it is optional as I have not seen any issues with uninitialized `kv_op`

Reply:
Remove this so that if we ever have issues related to not initializing the op variables with '0' then we need to catch them in our ut's and fix the m0_btree_put() to reset all the members in kv_op if it needs to.

Reply:
Ok.